STACK-UP-3/warriorz-backend
# #171784089 Edit User Profile Information

#### What does this PR do?
Enable user to update profile info
#### Description of Task to be completed?
- Create update user profile endpoint
- Create update user photo endpoint 
- Create get user profile endpoint
#### How should this be manually tested?
Give that the user has signed up (refer to signup feature); clone the repo
cd into it and run npm install then npm run dev.
using postman, send a patch request at localhost:5000/api/v1/profile with data in the body.
and to get user profile, send get request at localhost:5000/api/v1/profile with an authorization token in headers
Refer to the API docs by browsing: http://localhost:5000/api-docs/
Make sure you have Postgres database and create .env refer to .env.example

#### What are the relevant pivotal tracker stories?
[171784089](https://www.pivotaltracker.com/n/projects/2438874/stories/171784089)
